magnet/excitation-data/bo-corrector-ch-std.txt

```diff
@@ -41,7 +41,7 @@
 # corrector-vertical      | VKick > 0      | BXL  < 0    | -1.0        | I > 0
 # quadrupole (focusing)   | KL    > 0      | D1NL < 0    | -1.0        | I > 0
 # quadrupole (defocusing) | KL    < 0      | D1NL > 0    | -1.0        | I > 0
-# quadrupole (skew)       | KL    > 0      | D1SL > 0    | +1.0        | I > 0
+# quadrupole (skew)       | KsL   < 0      | D1SL > 0    | -1.0        | I > 0
```

Comment:
I used the label _KL_ for skew, just as for any other quad, to be in accordance to associated MA strength  PVs, which are also _KL_. @fernandohds564 , you think it is a good idea to introduce a new string here? The distinction between the two physical gradients are already taken into account (implicitly) by the distinct symbols _D1NL_ and _D2SL_. Should we carry the distinction to _KL_ / _KsL_ ? If so, I recommend also changing the MA skew quadrupole strength PVs to (_KsL_SP_, _KsL_RB_, _KsL_Mon_), for consistency.

Reply:
Ok @xresende, I changed the notation back to KL.